### PR TITLE
enhance: Give errors a name

### PR DESCRIPTION
**Pull request by ntucker**
### Motivation

Easier identification of errors

### CODE REVIEW

File: packages/core/src/state/NetworkManager.ts [hunk 1]
```diff
@@ -269,4 +277,6 @@ export default class NetworkManager implements Manager {
 
     return this.fetched[key];
   }
+
+  static ResetError = ResetError;
```
[the remark this answers is not in the document]
Comment (ljharb) [reply packages/core/src/state/NetworkManager.ts:281]: Very, why would it need to be a static property?

[the remark this answers is not in the document]
Comment (ntucker) [reply packages/core/src/state/NetworkManager.ts:281]: doesn't need to be; but might be convenient to access without having to do another import

[the remark this answers is not in the document]
Comment (ljharb) [reply packages/core/src/state/NetworkManager.ts:281]: that doesn't sound convenient, that sounds like a larger implicit API that will make it harder for consumers to know what they're using. explicit imports are a gift, not a burden :-)

**Review by ljharb — APPROVED** [review]
i defer to you, but i still think you should drop the static property

**Pull request merged**